style.css

```diff
@@ -121,10 +124,271 @@ h4 {
   list-style-type: none;
 }
 
+.connect-images {
+  display: flex;
+  width: 100%;
+  height: 100%;
+  margin-left: 5px;
+  margin-bottom: 13px;
+  list-style-type: none;
+}
+
 /* Vector */
 .vector {
   height: 25px;
   width: 40px;
   padding: 5px 5px;
   color: #505f79;
 }
+
+/* card containers */
+.card-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 16px;
+  gap: 12px;
+  width: 90%;
+  height: 100%;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  margin: 100px auto;
+}
+
+/* card-1 */
+.card-image {
+  width: 100%;
+  height: 100%;
+  border-radius: 8px;
+}
+
+/* Project titles */
+.project-title {
+  width: 70%;
+  height: 25%;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+  margin-top: 10px;
+  margin-right: 200px;
+  margin-left: -5px;
+}
+
+.texts {
+  display: flex;
+  justify-content: space-around;
```

Comment:
- [ ] Please use `justify-content: flex-start;` to be similar to the Figma design 👇🏽 

| Figma template | Your design |
|---|---|
| ![image](https://github.com/JoyTaribagshaw/My-Portfolio/assets/58642949/ef1cc6e4-c45b-45e2-9e73-dd74b434c409) | ![image](https://github.com/JoyTaribagshaw/My-Portfolio/assets/58642949/f6cfb63f-0d86-4a53-8324-7a32176f44cf) |